qdrant/grpc_client.go

```diff
@@ -84,3 +93,13 @@ func (c *GrpcClient) Snapshots() SnapshotsClient {
 func (c *GrpcClient) Close() error {
 	return c.conn.Close()
 }
+
+func getClientVersion() (string, error) {
+	packageName := "github.com/qdrant/go-client"
+	cmd := exec.Command("go", "list", "-m", "-f", "{{.Version}}", packageName)
```

Comment:
Worked for me locally.

Though I think this is brittle and could break if someone has renamed the package with something like `go mod edit -replace ...`.

Reply:
Unfortunately, I couldn't come up with another way of fetching client's version.